src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolution.cs

```diff
@@ -1543,6 +1562,90 @@ private static TypeSymbol GetParameterType(int argIndex, MemberAnalysisResult re
             return BetterResult.Neither;
         }
 
+        private static BetterResult BetterParamArrayTypeTieBreaker(TypeSymbol type1, TypeSymbol type2)
```

Comment:
This doesn't seem very rational. I would like us to consider whether we can break compat here for more rational overload resolution.